examples/gcp/backup/README.md

```diff
@@ -0,0 +1,127 @@
+# GCP Cloud Backup Examples
+
+The examples here show different ways of configuring the cloud backup CronJob to send backups
+in [GCP Cloud Storage](https://cloud.google.com/storage/docs)  
+and how to authenticate in GCP:
+
+* Using [ADC with Service Account's JSON key file](https://cloud.google.com/docs/authentication/set-up-adc-local-dev-environment#local-key).
+* Using [IAM Service Account impersonation](https://cloud.google.com/iam/docs/workload-identity-federation-with-kubernetes#use-service-account-impersonation) to grant access.
+
+## ADC with Service account keys
+
+⚠️ **Warning:** This example is not recommended. Instead of using long-term service account keys, consider using
+Workload Identity Federation(Linking the GCP Service Account to Kubernetes Service Account).
+
+The easiest and most straightforward way is to configure long-term credentials for the GCP Cloud Storage via an
+environment variable:
+
+Create the secret that contains the JSON key:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: gcp-sa-key
+type: Opaque
+data:
+  key.json: <BASE64_ENCODED_JSON_KEY>
+```
+
+Mount the secret and create an environment variable that references it:
+
+```yaml
+backup:
+  enabled: true
+  type: cloud
+  schedule: "@midnight"
+  cloud:
+    bucketUri: gs://<bucket>/<folder>/${BACKUP_NAME}
+
+extraEnv:
+  - name: GOOGLE_APPLICATION_CREDENTIALS
+    value: <json-key-reference>
+
+extraVolumes:
+  - name: gcp-key
+    secret:
+      secretName: gcp-sa-key
+
+extraVolumeMounts:
+  - name: gcp-key
+    mountPath: <mount-path>
```

Comment:
I guess you can add an example of where to be mounted and be read via GOOGLE_APPLICATION_CREDENTIALS ?? i.e. /secrets/key.json ?